learn/what_is_meilisearch/comparison_to_alternatives.md

```diff
@@ -9,12 +9,144 @@
 MeiliSearch has been designed to be a ready-to-go open source solution and deliver an intuitive and instantaneous search experience. As the amount of information available on the websites increases as time goes by, it is essential to provide users with the most relevant results. Since many closed source search engines and open source alternatives can be used, determining which one to install is critical and may not seem easy at first.
 
 The present article covers a comparison of alternatives to MeiliSearch, i.e., available search technologies to be considered when planning to implement a search bar.
+The [comparison table](/learn/what_is_meilisearch/comparison_to_alternatives.md#comparison-table) gives a good overview of how MeiliSearch is different from its alternatives.
```

Comment:
It may have been me only, but I had to use the following path `/learn/what_is_meilisearch/comparison_to_alternatives.md#comparison-table` to get my browser to scroll to the anchor.